src/package.json

```diff
@@ -29,9 +29,11 @@
     "protobufjs": "^7.2.4",
     "protobufjs-cli": "^1.1.2",
     "style-loader": "^3.3.3",
+    "ts-loader": "^9.4.4",
```

Comment:
thanks.
I also used ts-loader, but decided to cut it, because we already use `tsc` for tracker.